client-sqldelight-gradle-plugin/src/main/kotlin/app/cash/backfila/client/sqldelight/plugin/GenerateBackfilaRecordSourceQueriesTask.kt

```diff
@@ -159,7 +160,7 @@ abstract class GenerateBackfilaRecordSourceQueriesTask : DefaultTask() {
               .addParameter("start", keyType)
               .addParameter("end", keyType)
               .returns(recordQueryType)
-              .addStatement("return database.%L.getBatch(start, end)", queriesFunctionName)
+              .addStatement("return database.%L.%LGetBatch(start, end)", queriesFunctionName, lowerName)
```

Comment:
Is this preferred to `%LGetBatch`?
```suggestion
              .addStatement("return database.%L.%L(start, end)", queriesFunctionName, "${lowerName}GetBatch")
```